app.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const cors = require('cors');
+const passport = require('passport');
+const passportLocal = require('passport-local').Strategy;
+const cookieParser = require('cookie-parser');
+const session = require('express-session');
+const bodyParser = require('body-parser');
+
+//Runs express
+const app = express();
+
+// Middleware
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}))
+app.use(cors({
+    origin: "http://localhost:3000", // change to location of react app
+    credentials: true
+}))
+
+app.use(session({
+    secret: "secretcode", //needs to match in cookie parser
+    resave: true,
+    saveUninitialized: true
+}))
+
+app.use(cookieParser("secretcode")) //needs to match session secret
+
+app.use(passport.initialize());
+app.use(passport.session());
+
+
+//Routes
+require('./passportConfig')(passport);
+const loginController = require('./controllers/loginController');
+const registerController = require('./controllers/registerController');
+const getUserInfoController = require("./controllers/getUserInfoController");
+const getLoggedInController = require("./controllers/getLoggedInController");
+
+
+app.post("/user", registerController);
+app.post("/user/login", loginController);
+app.get("/user/loggedIn", getLoggedInController);
```

Comment:
It's not clear from the naming what this endpoint is supposed to return. The two options:
- boolean true/false whether the current user is logged in
- the details of the currently logged in user

You could rename it to either `currentUser` or `isLoggedIn` to make it clearer.

Reply:
Based on the controller I'm assuming the intention is for this to return the info of the currently logged in user. In that case I would also propose to change the name of the controller from `getLoggedInController` to `getLoggedInUserController`. This because the name of the controller is also ambiguous.